src/python/grpcio_tests/tests_py3_only/interop/Dockerfile.client

```diff
@@ -0,0 +1,25 @@
+FROM phusion/baseimage:master@sha256:65ea10d5f757e5e86272625f8675d437dd83d8db64bdb429e2354d58f5462750
+
+RUN apt-get update -y && \
+        apt-get install -y \
+            build-essential \
+            clang \
+            python3 \
+            python3-dev
+
+WORKDIR /workdir
+
+RUN ln -s /usr/bin/python3 /usr/bin/python
+RUN mkdir /artifacts
+
+COPY . .
+RUN tools/bazel build -c dbg //src/python/grpcio_tests/tests_py3_only/interop:xds_interop_client
+RUN cp -rL /workdir/bazel-bin/src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client* /artifacts/
+
+FROM phusion/baseimage:master@sha256:65ea10d5f757e5e86272625f8675d437dd83d8db64bdb429e2354d58f5462750
```

Comment:
This is a nice feature, TIL Docker has multi-stage builds. Just curious, can we further shrink the final image? E.g., use `python:3-alpine`

Reply:
Phusion is the smallest glibc image I know of. It's only around 80MB. I don't want to risk using musl libc and getting unexpected segfaults.